homeassistant/components/screenlogic/sensor.py

```diff
@@ -60,6 +60,16 @@ class ScreenLogicPushSensorDescription(
         state_class=SensorStateClass.MEASUREMENT,
         translation_key="air_temperature",
     ),
+    ScreenLogicPushSensorDescription(
+        subscription_code=CODE.STATUS_CHANGED,
+        data_root=(DEVICE.CONTROLLER, GROUP.SENSOR),
+        key=VALUE.STATE,
+        device_class=SensorDeviceClass.ENUM,
+        options=["unknown", "ready", "sync", "service"],
```

Comment:
Let's not have unknown as a valid option as that is an option by default. If the state is unknown we should return None

Reply:
"unknown" was just the enum label given to a possible value of 0 for the controller state provided by the pool controller, nothing to do with HA's unknown or unavailable states.
Valid values from the pool controller are supposed to be 1-3, but I've only ever seen a value of 1 or 3 in testing. I don't know if 0 could even ever be returned. I can just remove the unknown option.

Reply:
Oh but if we don't know it exists we should definitely remove it for now and just return None when that happens. This way if you ever get a 4, it's also unknown